solana/program_v2/programs/gateway_v2/src/instructions/gatekeeper/verify_pass.rs

```diff
@@ -53,8 +53,6 @@ pub struct PassVerify<'info> {
     #[account(
     seeds = [PASS_SEED, pass.subject.as_ref(), network.key().as_ref(), & pass.pass_number.to_le_bytes()],
     bump = pass.signer_bump,
-    // TODO!: Constraint is incorrect here (IDCOM-2232)
-    constraint = gatekeeper.can_access(& authority, GatekeeperKeyFlags::EXPIRE_PASS),
     mut
     )]
     pub pass: Box<Account<'info, Pass>>,
```

Comment:
This comment is on the wrong spot but github isn't letting me comment on line 65 `constraint = pass.gatekeeper == gatekeeper.key(),` should this still be there?

Reply:
@JulianSpring this is correct... this is that prevents a different gatekeeper other than the issuing one to recieve payment

Reply:
Maybe I am wrong, but this constraint implies that only the passes gatekeeper can verify? I thought any gatekeeper should be able to verify? Also shouldn't the gatekeeper doing the verification get the fee?

Reply:
Ah I see thanks!